lib/SIL/SILPrinter.cpp

```diff
@@ -1176,6 +1176,78 @@ class SILPrinter : public SILInstructionVisitor<SILPrinter> {
     *this << ") : ";
     *this << BI->getType();
   }
+
+  // SWIFT_ENABLE_TENSORFLOW
+  void visitSymbolicValue(SymbolicValue v) {
+    switch (v.getKind()) {
+    case swift::SymbolicValue::Integer:
+      *this << v.getIntegerValue();
+      return;
+    case swift::SymbolicValue::Float: {
+      SmallString<12> decimal;
+      v.getFloatValue().toString(decimal);
+      *this << decimal;
+      return;
+    }
+    case swift::SymbolicValue::String:
+      // TODO: Uncomment when `getStringValue` is implemented.
```

Comment:
Want to take this on as your next project? :-)

Reply:
Sure!